test/users.js

```diff
@@ -21,6 +22,14 @@ const someUser2 = {
   password: '12345678a'
 };
 
+const someAdministratorUser = {
+  firstName: 'Matias',
+  lastName: 'Perez',
+  email: 'matias.perez@wolox.com.ar',
+  password: '12345678a',
+  permission: enums.PERMISSION.ADMINISTRATOR
+};
+
 describe('/users POST', () => {
   it('should pass sign up, parameters are valid', () => {
     let response;
```

Comment:
no need for this variable

Reply:
This is the fix I made before in the other PR. It will get fixed when I merge with master.